src/NanoProtocol.cpp

```diff
@@ -0,0 +1,31 @@
+#include "NanoProtocol.h"
+#include <stdio.h>
+#include <FileIO.h>
+#include <g3log/g3log.hpp>
+NanoProtocol::NanoProtocol(const std::string& location) {
+   if (location.find("ipc") != std::string::npos) {
+      mProtocol = Protocol::IPC;
+      mLocation = location.substr(location.find("ipc://") + 6);
+      mOwner = !FileIO::DoesFileExist(mLocation);
+      LOG_IF(DEBUG, mOwner) << "i own the location: " << location;
```

Comment:
I think some, maybe most, of your DEBUG should be INFO. @KjellKod, do you have an opinion?